index.bs

```diff
@@ -2133,6 +2187,120 @@ and a [=boolean=] |fenced|:
 
 Note: The user agent may optionally include error details of any type in |body|["`error`"].
 
+<h3 id="attribution-debug-mode">Attribution debug mode</h3>
```

Comment:
I'm not sure that `mode` is the best terminology here. It seems easily confusable with something like the `local testing mode` that applies to the user agent as a whole, rather than for a particular registration.

Reply:
Updated to `Attribution debugging`, thanks.